Cargo.toml

```diff
@@ -19,14 +19,14 @@ exclude       = [
 
 [dependencies]
 log    = "0.3.1"
-nix    = { git = "https://github.com/carllerche/nix-rust", rev = "c4257f8a76" }
+nix    = "0.6.0"
 libc   = "0.2.4"
 slab   = "0.1.0"
 time   = "0.1.33"
 bytes  = "0.3.0"
 winapi = "0.2.1"
 miow   = "0.1.1"
-net2   = { version = "0.2.19", default-features = false }
+net2   = { version = ">=0.2.19,<=0.2.23", default-features = false }
```

Comment:
For better or for worse, the version of Cargo included w/ each version of Rust also needs to be supported on the same semver point release. If you break compatibility w/ Cargo you need to bump the semver..

I think net2 was fixed though